modules/ting_search/ting_search.module

```diff
@@ -226,9 +226,27 @@ function ting_search_form_search_block_form_alter(&$form, &$form_state, $form_id
   // information.
   $form['#action'] = url('search-blank');
 
+  // Adding content search option into ting_field_search dropdown.
+  if (!empty($form['ting_field_search'])) {
+    $node_search = array('node' => t('Search Content'));
+    $form['ting_field_search']['#options'] = $node_search + $form['ting_field_search']['#options'];
+    $form['#submit'][] = 'ting_search_content_search_submit';
+  }
+
   return $form;
 }
 
+/**
+ * Form submit callback for ting_field_search site content search option.
+ */
+function ting_search_content_search_submit(&$form, &$form_state) {
```

Comment:
This could also be handled in ting_field_search in `ting_field_search_search_block_submit()`, if you move the adding of the option to `ting_field_search_form_search_block_form_alter()` also. (See comment above).